### PR TITLE
Fixed get_file_data hash bug

### PR DESCRIPTION
**Pull request by cheediep**
$key in get_file_data is generated using full system path of module.
This causes a new entry to be added to the file_data field in the
database every time the folder path changes (it's a problem when deploying site to
new folders with version tags via Ansible). Overtime this field gets huge and causes site meltdown right after deployment when the field attempts to update for each module.

### CODE REVIEW

File: class.jetpack.php [hunk 1]
```diff
@@ -1819,8 +1819,10 @@ public static function get_module( $module ) {
 	 * Like core's get_file_data implementation, but caches the result.
 	 */
 	public static function get_file_data( $file, $headers ) {
+                //Get just the filename from $file (i.e. exclude full path) so that a consistent hash is generated
+                $fileName = basename($file);
```
Comment (samhotchkiss) [inline class.jetpack.php]: Just a few style issues:
1) We don't use camel case-- let's go with $file_name
2) These lines should be indented with tabs, not spaces
3) There should be spaces around $file `basename( $file )`

**Comment by samhotchkiss** [comment]
Hi @cheediep !  Thanks for submitting it-- there are a few style changes that need to be made, if you can do that, we should be able to get this in for 3.6

**Comment by cheediep** [comment]
Hi @samhotchkiss I've made the requested changes. Thanks!

**Comment by samhotchkiss** [comment]
Looks great, @cheediep -- I'm signing off on it for 3.6, which is entering beta next week.  As long as one other dev signs off, it'll be there :)

**Comment by dereksmart** [comment]
:+1:

**Pull request merged**